omim2obo/mondo_omim_genes_robot_tsv.py

```diff
@@ -0,0 +1,64 @@
+"""Create: ROBOT template of Mondo and OMIM gene relations: relational information for gene and disease classes"""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+
+from omim2obo.utils.utils import remove_angle_brackets
+
+
+ROBOT_SUBHEADER = {
+    'mondo_id': 'ID',
+    'hgnc_id': "SC 'has material basis in germline mutation in' some %",
+    'omim_disease_xref': '>A oboInOwl:source',
+    'source_code': '>A oboInOwl:source',
+    'omim_gene': '',
+}
+
+
+def mondo_omim_genes_robot_tsv(inpath: Union[Path, str], outpath: Union[Path, str]) -> pd.DataFrame:
+    """Create: ROBOT template of Mondo and OMIM gene relations"""
+    df = pd.read_csv(inpath, sep='\t')
+
+    # Remove the first character, a question mark (?), from each field in the header; an artefact of the SPARQL query.
+    df.rename(columns={col: col[1:] for col in df.columns if col.startswith('?')}, inplace=True)
+
+    # Add source_code column
+    df['source_code'] = 'MONDO:OMIM'
+
+    # Remove < and > characters from specified columns
+    uri_cols = ['mondo_id', 'hgnc_id', 'omim_gene']
+    for col in uri_cols:
+        df[col] = remove_angle_brackets(list(df[col]))
```

Comment:
Fine as-is, but I would do this more simply as:
```
for column in uri_cols:
    df[column] = df[column].str.replace('<', '').str.replace('>', '')
```

Reply:
Maybe you're right. That is simple and quick; maybe we don't need a function for this. That's the function btw that Harshad and I wrote in `mondo-ingest` and that Nico and I were talking about moving to `mondolib`.

It does a couple more minor things. But one difference I note is that it only removes these characters if they are at the beginning and end of the string, which is safer, though technically I don't expect that these would appear in the middle of a URI; they'd get serialized with some hex code.